src/abstractions/src/functions/KernelFunctionFromPrompt.ts

```diff
@@ -1,45 +1,20 @@
-import { ChatCompletionService, PromptExecutionSettings } from '../AI';
 import { Kernel } from '../Kernel';
-import { ChatMessageContent } from '../contents';
+import { ChatClient, ChatOptions } from '../chatCompletion';
 import { type FromSchema } from '../jsonSchema';
-import {
-  PassThroughPromptTemplate,
-  PromptTemplate,
-  PromptTemplateConfig,
-  PromptTemplateFormat,
-} from '../promptTemplate';
-import { AIService } from '../services';
-import { KernelArguments } from './KernelArguments';
+import { KernelFunctionFromPromptMetadata, PassThroughPromptTemplate, PromptTemplate, PromptTemplateFormat } from '../promptTemplate';
 import { KernelFunction } from './KernelFunction';
 
-export type PromptRenderingResult = {
-  renderedPrompt: string;
-  executionSettings?: PromptExecutionSettings;
-  AIService: AIService;
-};
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Would it be possible to define a type for this, so we can avoid the unused var linting error (and because it's used as a type)

Reply:
Sure I will look into it